releases_manager/releases_manager.py

```diff
@@ -1,11 +1,14 @@
 from github_manager import GithubManager
+import logging
 
 
 class ReleasesManager:
     def __init__(self):
         self.github_manager = GithubManager()
 
     def create_new_release(self, repo_name=None, major=False, minor=False, patch=False):
+
+        logging.info("Creates a new realease")
```

Comment:
Talvez fosse interessante logar as coisas aqui. Como o outro arquivo é um arquivo de API, nem sempre é o caso de logar quando algum metodo daqui for usado. Ou seja, ao invez de comentar no proprio metodo, talvez fosse interessante comentar embaixo da chamada nesse arquivo